GitCommands/Git/GitRevision.cs

```diff
@@ -26,6 +27,9 @@ public sealed class GitRevision : IGitItem, INotifyPropertyChanged
         public static readonly Regex Sha1HashShortRegex = new Regex(@"\b[a-f\d]{7,40}\b", RegexOptions.Compiled);
 
         private BuildInfo _buildStatus;
+        private const int CommitSummaryMaxLineLength = 150;
+        private const int CommitSummaryMaxNumberOfLines = 30;
+        private const int CommitSummaryWorstCaseLength = (CommitSummaryMaxNumberOfLines * (CommitSummaryMaxLineLength + 7)) + 5;
```

Comment:
Please add a comment to explain the magic numbers.

Reply:
done